web-components/src/components/combobox/ComboBox.ts

```diff
@@ -82,6 +82,7 @@ export namespace ComboBox {
     @property({ type: Boolean, attribute: "show-custom-error", reflect: true }) showCustomError = false;
     @property({ type: Boolean, attribute: "show-loader", reflect: true }) showLoader = false;
     @property({ type: Boolean, attribute: "show-selected-count", reflect: true }) showSelectedCount = false;
+    @property({ type: String, attribute: "popup-indicator-aria-hidden" }) popupIndicatorAriaHidden = "true";
```

Comment:
rename the prop from popup-indicator-aria-hidden to popup-chevron-aria-hidden

Reply:
done